src/apps/explorer/components/TokensTableWidget/PaginationTokensTable.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useContext } from 'react'
+import styled, { css } from 'styled-components'
+import { faChevronRight, faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { media } from 'theme/styles/media'
+
+import { Dropdown, DropdownOption } from 'apps/explorer/components/common/Dropdown'
+import { TokensTableContext } from './context/TokensTableContext'
+
```

Comment:
Could the pagination display and logic be somehow re-used between here and the orders page?
I have the impression that this is duplicated code.